_posts/2022-07-26-phone-number-capture-dataset.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Investigating Label Noise in intent classification datasets and fixing it
+date: 2022-07-26
+tags: [phone-number-capture-dataset]
+categories: [Machine Learning]
+image: assets/images/demo1.jpg
+layout: post
+authors: [anirudhthatipelli]
+latex: True
+fancy_audio_player: True
+---
+
+> This blog post is a dataset release for the Phone Number Entity capture task.
+
+# Introduction
+
+A bottleneck of a dialogue system is its ability to extract information from the utterances.  The information is extracted in the form of **frames**, that represents all the different types of intentions that the system can extract from user utterances and **slots**, that are the different type of possible values.
+
+![image](https://user-images.githubusercontent.com/16001446/180951003-545b10f0-3762-4aad-bc45-d602e4d1a872.png)
+For example, for the above conversation to book a flight, the frame will be of the type:
+
+![image](https://user-images.githubusercontent.com/16001446/180951715-d0169425-8f17-443e-a823-23dff1e5af46.png)
+# Motivation
```

Comment:
Can this be merged with the previous section ? `Introduction` isnt saying much really